client/state/happychat/selectors/get-geolocation.js

```diff
@@ -0,0 +1,8 @@
+/** @format */
+
+/**
+ * Returns the geo location of the current user, based happychat session initiation (on ip)
+ * @param {Object}  state  Global state tree
+ * @return {?String}        Current user geo location
+ */
+export default state => state.happychat.user.geoLocation || null;
```

Comment:
Maybe we could consider to use `lodash.get()` here as:
```
state => get( state, 'happychat.user.geoLocation', null )
```
so that this selector is more immune to state tree changes.

Reply:
Good catch! I did that for the other selectors but this one went unnoticed.